sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/ConsistentReadControl.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.models;
+
+import com.azure.storage.blob.BlobClient;
+
+/**
+ * Defines values for ConsistentReadControl.
```

Comment:
I think we should add a bit more here: "Defines values to indicate what strategy the sdk should use when reading from a blob to ensure the view of the data is consistent and not changed during the read."